k8s.gcr.io/images/k8s-staging-cluster-api-do/images.yaml

```diff
@@ -0,0 +1,3 @@
+- name: cluster-api-do-controller
+  dmap:
+    "sha256:": ["v0.3.0"]
```

Comment:
What is the process to get the SHA256? Should I first cut the release, build the image locally, and then past the SHA256? Is there some other process?

Reply:
You shouldn't actually need to list any images prior to having an image pushed to staging first.

Reply:
After you have a published staging image, you can get the sha either from the output of the job that is pushing the image, or you can look in the GCR registry through the console, for example the staging repo for cluster-api-provider-aws is viewable here: https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-aws/GLOBAL/cluster-api-aws-controller?project=k8s-staging-cluster-api-aws&gcrImageListsize=30 (may not be viewable if you are not in the group membership)

Reply:
That makes sense! Thanks for the explanation! I've dropped this file from the PR.